js/src/lib/mod.ts

```diff
@@ -110,3 +111,12 @@ export const newSpanId = (): TelemetryId => {
 export const newTraceId = (): TelemetryId => {
   return newTelemetryId();
 };
+export interface SpanFilter {
+  minimumDurationMicroseconds: Microseconds
+}
+
+export class Options {
+  spanFilter: SpanFilter = {
+    minimumDurationMicroseconds: 0
```

Comment:
IMO, we should set a higher minimum by default throughout the SDKs. Personally, I'd rather see less data and learn about configuration, vs. potentially sending huge traces to endpoints and causing a spike in cost for users. 

Something like 20 - 50 microseconds seems reasonable, but we should get some feedback

Reply:
I can start with 50 until we get more feedback